webpack/scenes/AlternateContentSources/ACSActions.js

```diff
@@ -78,6 +80,30 @@ export const refreshACS = (acsId, handleSuccess) => post({
   errorToast: error => acsErrorToast(error),
 });
 
+export const bulkRefreshACS = (params, handleSuccess) => post({
+  type: API_OPERATIONS.POST,
+  key: BULK_ACS_REFRESH_KEY,
+  url: api.getApiUrl('/alternate_content_sources/bulk/refresh'),
+  params,
+  handleSuccess: (response) => {
+    renderTaskStartedToast(response?.data);
+    return handleSuccess();
+  },
+  errorToast: error => __(`Something went wrong while refreshing alternate content sources: ${getResponseErrorMsgs(error.response)}`),
```

Comment:
We cannot translate dynamic values like getResponseErrorMsgs(error.response)
This should work except for translating error response message.
```suggestion
  errorToast: error => __(`Something went wrong while refreshing alternate content sources: `) + getResponseErrorMsgs(error.response),
```